dynatrace-aws-logs.sh

```diff
@@ -127,9 +143,22 @@ arguments:
   if [ -z "$TARGET_API_TOKEN" ]; then echo "No --target-api-token"; print_help_deploy; exit 1; fi
   if [ -z "$REQUIRE_VALID_CERTIFICATE" ]; then REQUIRE_VALID_CERTIFICATE="false"; fi
   if [ -z "$STACK_NAME" ]; then STACK_NAME=$DEFAULT_STACK_NAME; fi
+  if [ -z "$USE_EXISTING_ACTIVE_GATE" ]; then echo "No --use-existing-active-gate"; print_help_deploy; exit 1; fi
 
   if [[ "$REQUIRE_VALID_CERTIFICATE" != "true" ]] && [[ "$REQUIRE_VALID_CERTIFICATE" != "false" ]];
     then echo "Invalid value for parameter --require-valid-certificate. Provide 'true' or 'false'"; print_help_deploy; exit 1; fi
+  if [[ "$USE_EXISTING_ACTIVE_GATE" != "true" ]] && [[ "$USE_EXISTING_ACTIVE_GATE" != "false" ]];
+    then echo "Invalid value for parameter --use-existing-active-gate. Provide 'true' or 'false'"; print_help_deploy; exit 1; fi
+  if [[ "$USE_EXISTING_ACTIVE_GATE" == "false" ]]; then
+    if [ -z "$PAAS_TOKEN" ]; then echo "No --paas-token (required when --use-existing-active-gate=false)"; print_help_deploy; exit 1; fi
```

Comment:
1. minor, I'd merge if 152 & 153
2. For me this message is a bit confusing (existing=false) doesn't convey real reason why in this case it is required.  Not sure if it's way better, but something in the direction of: 
`--paas-token is required to allow installation a new ActiveGate. To use an existing one change --use-existing-active-gate=true`
But this sounds already more like part of a doc then command line message .So maybe just "no --paas-token" like with the other messages and then help is printed either way.

Reply:
1. Right, will fix 
2. So basically just remove this explanation in parens? Sure. Yes, it is more explained in help and help is printed in this case